moduleframework/common.py

```diff
@@ -348,7 +288,7 @@ def get_profile():
     :return: str
     """
 
-    return os.environ.get('PROFILE') or MODULE_DEFAULT_PROFILE
+    return os.environ.get('PROFILE') or conf["modularity"]["default_profile"]
```

Comment:
the same.

Reply:
@jscotka This returns, either environment variable or value from `conf` or None. Is it requested?

Reply:
yes this is expected. or this also can cause traceback in case some user break this value from config. It is expected